gem/rbi-central.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.metadata["allowed_push_host"] = "https://rubygems.org"
 
   spec.add_dependency("json-schema", ">= 3.0.0")
+  spec.add_dependency("bigdecimal", ">= 3.0.0")
```

Comment:
We were getting the following deprecation warning:

```
/Users/emilysamp/.gem/ruby/ruby-3.3.0/gems/json-schema-4.3.0/lib/json-schema/validator.rb:3: warning: bigdecimal was loaded from the standard library, but will no longer be part of the default gems since Ruby 3.4.0. Add bigdecimal to your Gemfile or gemspec. Also contact author of json-schema-4.3.0 to add bigdecimal into its gemspec.
```

I'll open an issue with json-schema but it's not super actively maintained (right now you can't even  develop the gem on Ruby 3.3 because of *other* deprecation warnings) so I figured we should fix it on our end to unblock ourselves.